mix.exs

```diff
@@ -8,7 +8,8 @@ defmodule Onigumo.MixProject do
       elixir: "~> 1.10",
       start_permanent: Mix.env() == :prod,
       deps: deps(),
-      escript: escript()
+      escript: escript(),
+      elixirc_paths: elixirc_paths(Mix.env())
```

Comment:
`Mix.env()` is called twice. We can save the repeated call by storing the returned value to a variable.